bayer2ppm

```diff
@@ -26,7 +26,7 @@ import imgio            # pip install imgio
 import rawpy            # pip install rawpy
 import rawpipe          # pip install rawpipe
 import argv             # local import: argv.py
-
+import matplotlib.pyplot as plt # pip install matplotlib
```

Comment:
Perhaps use a separate tool for viewing the Y-data? I'd like to avoid having matplotlib as a dependency.

Reply:
Yes, agree, let's remove this dependency. 

Meanwhile is `rawpy` really needed to be 0.14 or newer? My Windows 10's WSL Ubuntu seems does not have it out of the box: 

```
Collecting rawpy>=0.14.0 (from -r requirements.txt (line 5))
  Could not find a version that satisfies the requirement rawpy>=0.14.0 (from -r requirements.txt (line 5)) (from versions: 0.1.0, 0.2.0, 0.3.0, 0.3.1, 0.3.2, 0.3.3, 0.3.4, 0.3.5, 0.4.0, 0.5.0, 0.5.1, 0.6.0, 0.7.0, 0.8.0, 0.8.1, 0.9.0, 0.10.0a1, 0.10.0, 0.10.1, 0.11.0a1, 0.11.0, 0.12.0a1, 0.12.0, 0.13.0, 0.13.1)
No matching distribution found for rawpy>=0.14.0 (from -r requirements.txt (line 5))
```

Reply:
I think there was a bugfix in rawpy 0.14 that's required, can't recall what it was though. Version 0.15.0 is available via pip: https://pypi.org/project/rawpy/ and I have it on my WSL Ubuntu. Try `pip3 install --upgrade rawpy`.